src/types/form_issue.ts

```diff
@@ -0,0 +1,14 @@
+import { Document, Model } from 'mongoose'
+import { FormIssueBase } from 'shared/types'
+import type { Merge } from 'type-fest'
+
+import { IFormSchema } from './form'
+
+export type IFormIssue = Merge<FormIssueBase, { formId: IFormSchema['id'] }>
+
+export interface IFormIssueSchema extends IFormIssue, Document {
+  created?: Date
+  lastModified?: Date
```

Comment:
Is it necessary to redeclare these 2 properties? 🤔 

`IFormIssueSchema` extends `IFormIssue`, itself created from `FormIssueBase`, which already contains the 2 fields.

Reply:
Honestly, all other schemas are defined the same way, where the Base class also has a created and lastModified. But you are right, it seems to be just duplicated. I have tested it and the flow still works